browser/src/FederationPage.tsx

```diff
@@ -0,0 +1,236 @@
+  },
+  {
+    country: 'China',
+    samples: '10,000',
+    source: 'China Kadoorie Biobank (>500K)',
+    link: 'https://gnomad.org.cn',
+    lead: 'Xiao Li',
+  },
+  {
+    country: 'Europe',
+    samples: 'TBD',
+    source: 'European Genome Phenome Archive',
+    link: 'https://ega-archive.org/',
+    lead: 'Abeer Fadda',
+  },
+  {
+    country: 'Europe',
+    samples: '30,000',
+    source: 'European Reference Network for Rare Neurological Diseases',
+    link: 'https://www.ern-rnd.eu/',
+    lead: 'Holm Graessner',
+  },
+  {
+    country: 'Germany',
+    samples: '10,000',
+    source: 'Clinical sequencing',
+    link: 'https://www.igsb.uni-bonn.de/en',
+    lead: 'Peter Krawitz',
+  },
+  {
+    country: 'Germany',
+    samples: 'TBD',
+    source: 'The German Human Genome-Phenome Archive (GHGA)',
+    lead: 'Drew Behrens',
+  },
+  {
+    country: 'Japan',
+    samples: '100,000',
+    source: 'Tomoku Medical Megabank Organization',
+    link: 'https://www.megabank.tohoku.ac.jp/english/',
+    lead: 'Soichi Ogishima',
+  },
+  {
+    country: 'Qatar',
+    samples: '25,000',
+    source: 'Qatar Biobank',
+    link: 'https://www.qphi.org.qa/',
+    lead: 'Chadi Saad',
+  },
+  {
+    country: 'Singapore',
+    samples: '10,000',
+    source: 'SG10K_Health data',
+    link: 'https://www.a-star.edu.sg/gis/our-science/precision-medicine-and-population-genomics/npm',
+    lead: 'Maxime Hebrard',
+  },
+  {
+    country: 'Taiwan',
+    samples: '1,500',
+    source: 'Taiwan Biobank',
+    link: 'https://genomes.tw/',
+    lead: 'Jacob Shujui Hsu',
+  },
+  {
+    country: 'UK',
+    samples: '50,000',
+    source:
+      'Avon Longitudinal Study of Parents and Children (ALSPC), Born in Bradford, Millennium Cohort Study (MCS), Fenland Study',
+    link: 'https://www.sanger.ac.uk/',
+    lead: 'Vivek Iyer',
+  },
+]
+
+export default () => (
+  <InfoPage>
+    <DocumentTitle title="Federated gnomAD" />
+    <PageHeading
+      // @ts-expect-error
+      id="federated-gnomad"
+    >
+      Federated gnomAD
+    </PageHeading>
+
+    <MarkdownContent dangerouslySetInnerHTML={{ __html: federationContent.html }} />
+
+    <br />
+
+    <CenteredContainer>
+      <img
+        src={FederatedGnomadMap}
+        alt="Locations of federated gnomAD data sources on a world map."
+        width="70%"
+      />
+    </CenteredContainer>
+
+    <br />
+
+    <CenteredContainer>
+      <StatsTable>
+        <thead>
+          <StatsTableHeaderRow>
+            <th>Country</th>
+            <th>Samples</th>
+            <th>Source</th>
+            <th>Lead</th>
+          </StatsTableHeaderRow>
+        </thead>
+        <StatsTableBody>
+          {datasets.map((dataset) => {
+            return (
+              <tr key={dataset.source}>
+                <td>{dataset.country}</td>
+                <td>{dataset.samples}</td>
+                <td>
+                  {dataset.link ? (
+                    // @ts-expect-error
```

Comment:
...whereas I'd guess this particular `ts-expect-error` is due to the known issue with elements of GBTK not having `children` declared?

Reply:
Yep :(